### PR TITLE
Only use downloaded images

### PR DESCRIPTION
**Pull request by chrise96**
Some panoramic images fail to download from CloudVPS and are therefore not saved in the storage acc

### CODE REVIEW

File: retrieve_images.py [hunk 1]
```diff
@@ -70,18 +70,24 @@ def download_panorama_from_cloudvps(
...
         print(f"Timeout Error: Failed for panorama {panorama_id}:\n{e}")
+        return ""
     except requests.exceptions.ConnectionError as e:
         # In the event of a network problem (e.g. DNS failure, refused connection, etc),
         # Requests will raise a ConnectionError exception.
         print(f"Connection error: Failed for panorama {panorama_id}:\n{e}")
+        return ""
     except requests.exceptions.RequestException as e:
         print(f"Unknown Error: Failed for panorama {panorama_id}:\n{e}")
+        return ""
```
Comment (epureanudiana) [inline retrieve_images.py:90]: you can put them all together like this:
```
 except (
        requests.exceptions.HTTPError,
        requests.exceptions.Timeout,
        requests.exceptions.ConnectionError,
        requests.exceptions.RequestException,
    ) as e:
        print(f"Error: Failed for panorama {panorama_id}:\n{e}")
        return ""
```

File: retrieve_images.py [hunk 2]
```diff
@@ -203,12 +209,15 @@ def get_pano_ids(start_date_dag_ymd: str) -> Any:
...
     local_retrieved_images_path = "retrieved_images"
+    dl_pano_ids = []
     for pano_id in pano_ids:
-        download_panorama_from_cloudvps(
+        dl_pano_id = download_panorama_from_cloudvps(
             datetime.strptime(start_date_dag_ymd, "%Y-%m-%d"),
             pano_id,
             local_retrieved_images_path,
         )
+        if dl_pano_id:
```
Comment (epureanudiana) [inline retrieve_images.py:219]: isn't` dl_pano_id` always be true? isn't `""==True`?

Comment (chrise96) [reply retrieve_images.py:219]: https://stackoverflow.com/questions/9573244/how-to-check-if-the-string-is-empty

Comment (epureanudiana) [reply retrieve_images.py:219]: ah, thanks

**Comment by epureanudiana** [comment]
Looks good, I find the try-catch block good enough for this

File: retrieve_images.py [hunk 3]
```diff
@@ -147,7 +153,7 @@ def get_pano_ids(start_date_dag_ymd: str) -> Any:
 
     saClient = StorageAzureClient(secret_key="data-storage-account-url")
 
-    development = True
+    development = False
```
Comment (chrise96) [inline retrieve_images.py:156]: @thomasbrockmeier-ams Misschien handig om deze op False te zetten nu omdat we al naar PRD gaan?

Comment (chrise96) [reply retrieve_images.py]: Teruggezet naar False. Otherwise, 15.000 images will be downloaded and processed for 2020-05-08 on test, acc and prod

**Pull request merged**